### DISCUSSION TITLE
About the height limit (5) of status area

### DISCUSSION THREAD

**Discussion by x1a0** [Blightmud/Blightmud#585] [Q&A] [answered]
Hi. Here comes another question:)

I want to print the "map" data in status area. In the Mud I play, BatMud, that's 13 lines for text.
Each time I got these lines, I use trigger to grab them and put them in status area.
There will be some other data/lines as well, such as visualized exits (3 lines).

In short, I'd need more lines in the area. I modified the code and change the limit to 20. So far everything looks good.

My question is... why having such a limit? I am just afraid at some point I'd hit some problem that I don't see now.

Thanks.

**Comment by LiquidityC** [comment, marked as answer]
Not sure why really. It just seemed like a reasonable number at the time of writing the code.

For your particular case I'd recommend taking a look at sockets and using tmux to split your terminal. That's what I do for info screens and maps 😉. (doesn't have to be tmux, any option for showing multiple terminals should do fine).

**Comment by x1a0** [comment]
Yeah saw the socket feature in docs and tried a bit. Haven't spent much time on it to make some real use.

I like tmux and use it daily so good to know that you have setup using it:)
What I concerned about tmux is the panes' size, mainly height, could change when the whole window size changes. When that happens I might need to go to each pane and resize them probably - which feels like an small but annoying trouble :sweat_smile: 
But maybe it's not as bad as I thought - will give it a try.
